Joedmin-hw3/Joedmin-hw3/Program.cs

```diff
@@ -0,0 +1,153 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+
+namespace Joedmin_hw3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            try
+            {
+                Console.Write("What do you want to do? login/register/exit: ");
+                string answer = Console.ReadLine();
+
+                // Exit
+                if (String.Equals(answer, "exit", StringComparison.OrdinalIgnoreCase))
+                {
+                    Process.GetCurrentProcess().Kill();
+                }
+
+                // Login
+                else if (String.Equals(answer, "login", StringComparison.OrdinalIgnoreCase))
+                {
+                    using (var stream = new StreamReader("Users.txt"))
```

Comment:
You never use `stream` as best I can tell.

Reply:
what? I've been told like it by Kai, if I remember correctly.